### DISCUSSION TITLE
How to modify the metrics-server service

### DISCUSSION THREAD

**Discussion by 983183947** [k3s-io/k3s#8065] [Q&A] [answered]
I want to modify the InternalIP priority of the metrics-server service, but once I restart k3s after the modification, it will be automatically restored.

**Comment by brandond** [comment, marked as answer]
See https://docs.k3s.io/installation/packaged-components#packaged-components:
> Manifests for packaged components are managed by K3s, and should not be altered. The files are re-written to disk whenever K3s is started, in order to ensure their integrity.

If you want to deviate from the packaged configuration, you should start k3s with `--disable=metrics-server` to disable the packaged deployment, and provide your own.
